### PR TITLE
[SC-160] Add contribution guidelines

### PR DESCRIPTION
**Pull request by barrutko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,3 +10,12 @@ Forwarders are contracts on the host domain, where the governance of the protoco
 Executors serve as admins of the bridged domain instances of the protocol. <br>They are responsible for storying the queue of proposals passed from the host domain governance and their execution. <br> They manage bridged domain protocol instance using standard payload pattern.
 ## ✍️ Architecture Diagram
 ![Architecture Diagram](/diagram.png)
+## 🤝 Contribution Guidelines
+In order to add governance relay infrastructure for a new domain, perform the following steps:
+1. Go to [XChain Helpers](https://github.com/marsfoundation/xchain-helpers) repository and add a proper domain helper abstracting away the process of passing messages between host domain and your bridged domain.
+2. Add proper `CrosschainForwarder` to the `/src/forwarders` directory and proper `BridgeExecutor` to the `/src/executors` directory. Follow currently used naming convention.
+3. If they are any dependencies required, copy them into `/src/dependencies/<YOUR_DOMAIN_NAME>` directory.
```
Comment (hexonaut) [inline README.md]: `they` -> `there` in second word

File: README.md [hunk 2]
```diff
@@ -10,3 +10,12 @@ Forwarders are contracts on the host domain, where the governance of the protoco
...
+4. Add a new test file for your domain to the `/test` directory. Inherit `CrosschainTestBase` and add tests specific to your domain to the test suite. All of the tests have to pass. Follow linting and naming convention used in other test files.
+5. Use proper labeling for your open PR (always set adequate priority and status)
+6. Get an approving review from at least two of three designated reviewers - @hexonaut, @lucas-manuel and @barrutko
```
Comment (hexonaut) [inline README.md]: We prefer the teams to mostly review their own work, but I would say they need at least 1 review from us (2 total).

**Review by hexonaut — CHANGES_REQUESTED** [review]
Some small changes.

**Review by hexonaut — DISMISSED** [review]
LGTM

**Review by lucas-manuel — DISMISSED** [review]
LGTM

**Review by hexonaut — APPROVED** [review]

**Review by lucas-manuel — APPROVED** [review]

**Pull request merged**